docker/ci/config/jdk-setup.sh

```diff
@@ -23,12 +23,14 @@ case "${ARCH}" in
        JDKS+="d10efb2afad3ed3d7bac9d3249cea77928aca6acb973cac0f90a2dd3606a3533@https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u332-b09/OpenJDK8U-jdk_aarch64_linux_hotspot_8u332b09.tar.gz "
        JDKS+="999fbd90b070f9896142f0eb28354abbeb367cbe49fd86885c626e2999189e0a@https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.15%2B10/OpenJDK11U-jdk_aarch64_linux_hotspot_11.0.15_10.tar.gz "
        JDKS+="2e3c19c1707205c6b90cc04b416e8d83078ed98417d5a69dce3cf7dc0d7cfbca@https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.3%2B7/OpenJDK17U-jdk_aarch64_linux_hotspot_17.0.3_7.tar.gz "
+       JDKS+="9b5de40b0f6fe0ab32e8d035720dbbc87bf41b758ed67351ad781ca6505f5294@https://github.com/adoptium/temurin19-binaries/releases/download/jdk-19%2B36/OpenJDK19U-jdk_aarch64_linux_hotspot_19_36.tar.gz "
```

Comment:
19.0.1+10 for aarch64 is not yet published

Reply:
That is ok, could you @ again once it is out.
We will then merge this PR.

Note this probably will take a while because all the related `current` runner that uses this jdk script would need to be updated.

Thanks.

Reply:
BTW I am also adding 19.0.1+10 in Jenkins so let's wait a bit.

Reply:
Jenkins is  x64 or aarch64? x64 is fine btw

Reply:
Both

Reply:
@peterzhuamazon JDK 19.0.1+10 `aarch64` bits released today, pull request updated